package.json

```
@@ -1629,6 +1640,10 @@
 				{
 					"command": "code-for-ibmi.sortMembersByName",
 					"when": "never"
+				},
+				{
+					"command": "code-for-ibmi.cl.runSelected",
+					"when": "code-for-ibmi:connected"
```

Only shows in palette if displaying a cl
```suggestion
					"when": "code-for-ibmi:connected && editorLangId === cl"
```